app/src/main/java/com/example/appempty/ViewModel/UserListViewModel.kt

```diff
@@ -33,8 +34,8 @@ class UserListViewModel {
             ) {
                 if (!response.isSuccessful) throw Throwable("error1")
 
-                val result: UserProfile = response.body()?.results?.firstOrNull() ?: UserProfile()
-                lista.value = result
+                val result: List<UserProfile>? = response.body()?.results
```

Comment:
You do not want to change anything in this class. It is not been used on the app. You can remove it if you are not planning to use it.

Reply:
Esta clase no la estoy "usando" por qué no me funciona. Ajusté el código de ListaFragment.kt   para usar UserViewModel.kt en mientras tenga la clase UserListViewModel disfuncional. Eso era en lo que trabajaba ayer.

Reply:
Lo que quise decir s que no tiens que modificar nada aqui. porqu eno se esta usando... asi que elimina esos cambios.

Reply:
Deleted Class